packages/langium-railroad/src/grammar-railroad.ts

```diff
@@ -39,16 +39,30 @@ export interface GrammarDiagramOptions {
     javascript?: string;
 }
 
+function styling(options?: GrammarDiagramOptions) {
+    return `<style>
+    ${defaultCss}
+    </style>${options?.css ? `
+    <style>
+    ${options.css}
+    </style>` : ''}`;
```

Comment:
Suggestion: This kind of string generation leads to unexpected indentation on every line after the first. Even though it's just HTML, we try to keep the generated code reasonably formatted. 
```suggestion
    return expandToString`
    <style>
    ${defaultCss}
    </style>${options?.css ? `
    <style>
    ${options.css}
    </style>` : ''}`;
```

Reply:
Now using `expandToString` as suggested.